tests/run_test.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
```

Comment:
Does this script have to be explicitly for Python3? We should try to support both Python 2.7 and 3+.

Reply:
Right now it requires Python three because of how I print to stderr, but I'll change that.

Reply:
Also, my use of `subprocess`.